templates/namespace/namespace.py

```diff
@@ -38,11 +38,20 @@ def _monitor(self):
         except StateCheckError:
             return
 
+        def _reinstall():
```

Comment:
I don't see why we create a function for this, we don't reuse it anywhere, you can just put these 2 lines, instead of the lie 54